### ISSUE TITLE
puppet parser validate

### ISSUE THREAD

**Issue by khaefeli** [croomes/bamboo-puppet-plugin#11]
we're using the plugin in production and would really appreciate if we could use a `puppet parser validate` for our manifests with this tests.

I see this feature it's "upcoming" - do you know already when? 

cheers

**Comment by croomes** [comment]
I'm not using Puppet anymore so this isn't something I still plan to add myself.  I'd welcome contributions, but for now I've removed the "coming-soon" plans from the marketplace readme.

**Comment by khaefeli** [comment]
@croomes I'd like to contribute - there is also a fix needed for bamboo 6.4
would you handover the plugin to the community? :)

**Comment by croomes** [comment]
yes - I've already added one co-maintainer and happy to add more.
